asdf/conftest.py

```diff
@@ -1,18 +1,13 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 # -*- coding: utf-8 -*-
-# this contains imports plugins that configure py.test for asdf tests.
-# by importing them here in conftest.py they are discoverable by py.test
+
+# This contains pytest fixtures used in asdf tests.
+# by importing them here in conftest.py they are discoverable by pytest
 # no matter how it is invoked within the source tree.
 
-import os
 
 import pytest
 
-from .extern.RangeHTTPServer import RangeHTTPRequestHandler
```

Comment:
Is this just a test dependency?  Doesn't seem worth vendorizing just for that.

Reply:
Yes, a dependency circuitously for `test_generic_io.py`.  I suspect something like `requests-mock` would be more appropriate.

Reply:
I'll make an issue for this